### DISCUSSION TITLE
Pin latest image to the top when viewing docker packages

### DISCUSSION THREAD

**Discussion by NGPixel** [community/community#11705] [Actions]
When using the GitHub Packages docker registry, the docker image page shows the 5 most recently tagged image versions and the command to use to fetch the most recent one above.

This is very problematic when dev images are pushed and the "latest" stable image version is no longer visible on that page at all. The example command at the top also points to the most recent one, regardless of whether it's a stable release or not. Some users could be tempted to use that command directly, using an unstable dev image instead of the stable "latest" one.

Please add an option to pin the image version with the tag "latest" to the top of the list and to make the example command above it use that image version as well.

**Comment by HB9HIL** [comment]
This would be awesome. Was this ever considered... ?
